src/Controller/Backend/BackendBase.php

```diff
@@ -42,6 +42,14 @@ protected function render($template, array $context = [], array $globals = [])
             $context = ['context' => $context];
         }
 
+        /** @deprecated Deprecated since 3.0, to be removed in 4.0. */
+        $oldGlobals = $this->app['twig']->getGlobals();
+        foreach (['user', 'users', 'bolt_version'] as $key) {
+            if (!isset($context['context'][$key])) {
+                $context['context'][$key] = $oldGlobals[$key];
+            }
+        }
```

Comment:
Idk. Where are they used? Maybe not `users`, but I could see `user` being useful. Maybe a function would be better though that way access can be checked when requested?